homeassistant/components/vicare/button.py

```diff
@@ -54,6 +57,15 @@ async def async_setup_entry(
     entities = []
 
     for description in BUTTON_DESCRIPTIONS:
+        try:
+            description.value_getter(api)
```

Comment:
Is this async safe, ie no blocking I/O, locking or sleeping?

Reply:
Thanks @MartinHjelmare , I need to keep more attention to this. Looking at the upstream code it looks like it ends with a synchronous HTTP request (blocking) @oischinger can you confirm?

Reply:
Sorry guys, you're absolutely right. Pyvicare caches the results but when out of sate this call could potentially end up in an http call.
I'll create a late review PR later today.

Thanks for the review!

Reply:
here it is: https://github.com/home-assistant/core/pull/79458